Index: DB_active_rec.php
===================================================================
--- DB_active_rec.php	(revision 1722)
+++ DB_active_rec.php	(working copy)
@@ -489,9 +489,10 @@
 	 * @param	array	The values searched on
 	 * @return	object
 	 */
-	function where_in($key = NULL, $values = NULL)
+	function where_in($key = NULL, $values = NULL, $escape = NULL)
 	{
-		return $this->_where_in($key, $values);
+		$escape = (is_bool($escape)) ? $escape : TRUE;
+		return $this->_where_in($key, $values, FALSE, 'AND ', $escape);
 	}
 	
 	// --------------------------------------------------------------------
@@ -562,7 +563,7 @@
 	 * @param	string	
 	 * @return	object
 	 */
-	function _where_in($key = NULL, $values = NULL, $not = FALSE, $type = 'AND ')
+	function _where_in($key = NULL, $values = NULL, $not = FALSE, $type = 'AND ', $escape = TRUE)
 	{
 		if ($key === NULL OR $values === NULL)
 		{
@@ -578,7 +579,14 @@
 
 		foreach ($values as $value)
 		{
-			$this->ar_wherein[] = $this->escape($value);
+			if ($escape === TRUE)
+			{
+				$this->ar_wherein[] = $this->escape($value);
+			}
+			else
+			{
+				$this->ar_wherein[] = $value;
+			}
 		}
 
 		$prefix = (count($this->ar_where) == 0) ? '' : $type;
